packages/react-lowcode/src/codegen/list.ts

```diff
@@ -20,12 +21,38 @@ export async function addColumn(typesSourceCode: string,
     options: InsertOptions): Promise<string | undefined>{
         
     const property: Property = getEntityProperty(typesSourceCode, options.property, options.entityName)[0]
+    const code = await io.readFile(sourceCode.fileName)
     let generatedSource = undefined
+    let isGrommet = false
```

Comment:
simplify like this please
`const isGrommet = code && isGrommetRepository(code)`
and write it one/two lines below